src/VisualStudio/Core/Def/Implementation/DesignerAttribute/VisualStudioDesignerAttributeService.cs

```diff
@@ -118,16 +118,15 @@ private async Task StartWorkerAsync()
 
             // Pass ourselves in as the callback target for the OOP service.  As it discovers
             // designer attributes it will call back into us to notify VS about it.
-            _connection = await client.CreateConnectionAsync(
-                WellKnownServiceHubService.RemoteDesignerAttributeService,
-                callbackTarget: this, cancellationToken).ConfigureAwait(false);
+            var connection = await client.CreateConnectionAsync<IRemoteDesignerAttributeService>(callbackTarget: this, cancellationToken).ConfigureAwait(false);
```

Comment:
still would like docs on if it's intentional that we are creating a field, but only assigning at the end of hte work.